src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs

```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
         /// <value>A string representing the information for 360 content, or null if the information does not exist.</value>
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
+        /// </summary>
+        /// <value>
+        /// 0:None(There's no stitched 360 content metadata), 1:Non-stitched content, 2:stitched content
+        /// </value>
+        /// <since_tizen> 12 </since_tizen>
+        /// <value>A string representing the information for stitched 360 content, or null if the information does not exist.</value>
```

Comment:
The `<value>` tag has been added twice, please use only one.

Reply:
I'll fix it.